blocks/init/src/blocks/components/paragraph/components/paragraph-toolbar.js

```diff
@@ -4,24 +4,20 @@ import { Fragment } from '@wordpress/element';
 
 export const ParagraphToolbar = (props) => {
   const {
-    paragraph: {
-      styleAlign,
-      removeStyle,
-    },
+    styleAlign,
+  } = props.paragraph;
+
+  const {
     onChangeStyleAlign,
   } = props;
```

Comment:
I wonder if this could have all been written like:

```js
export const ParagraphToolbar = ({paragraph, onChangeStyleAlign}) => {
  const { styleAlign } = paragraph;
}
```

?